packages/subcommands/package.json

```diff
@@ -0,0 +1,59 @@
+{
+	"name": "@sapphire/subcommands",
```

Comment:
```suggestion
	"name": "@sapphire/plugin-subcommands",
```

Reply:
Resolved in 6ea0a87491b4571304bf8069df488230e4a2b379